### ISSUE TITLE
How to add class from css framework?

### ISSUE THREAD

**Issue by rzwibowo** [TerryZ/v-suggest#16]

**Comment by TerryZ** [comment]
Try to add `class` attribute directly.

```vue
<v-suggest :data="data" class="the class you want">
</v-suggest>
```

**Comment by rzwibowo** [comment]
> Try to add `class` attribute directly.
> 
> ```
> <v-suggest :data="data" class="the class you want">
> </v-suggest>
> ```

I've been playing around in your codepen example and the result is like the screenshot I attached on my first edited issue comment

**Comment by TerryZ** [comment]
`v-suggest` dom elements structure like below:
![image](https://user-images.githubusercontent.com/854406/74703350-df41ab80-5247-11ea-9d5e-f4d7716131a5.png)

You have to write some css style to overwrite the original css content

```scss
/* the css content in your page */
div.v-suggest {
  input[type=text] {
    /* the css content you want to customize */
  }
}
```

**Comment by rzwibowo** [comment]
Yup, it's what I planned to try before. With your answer, now I know what specific selector I should overwrite. Thanks for your kind response!
